tests/regression/56-asm/03-basic-discard-global.c

```diff
@@ -0,0 +1,12 @@
+#include <assert.h>
+
+int x = 0;
+
+int main()
+{
+  asm("nop");
+
+  assert(x == 0); // UNKNOWN
```

Comment:
It would make sense to have additional tests for the configurable options you implement, e.g. you might add a test for what's currently `sem.asm.basic-preserve-global`.